README.md

```diff
@@ -1,3 +1,11 @@
+# Encointer Testnet Cantillon branch 
+Encointer-node branch to build a Testnet Cantillon with the encointer [community-sidechain](https://github.com/encointer/community-sidechain).
```

Comment:
Please change that to: `Encointer-Testnet Community Sidechain Branch` and on the second line you can just remove the `Cantillon`